src/screens/Forum/ForumHomeScreen.js

```diff
@@ -3,9 +3,10 @@ import {
   Text, View, ScrollView, StyleSheet,
```

Comment:
The reason why we want to remove this file from the PR is because it contains code that we don't want in our final product!